### DISCUSSION TITLE
Touch Typing learning - websites and software

### DISCUSSION THREAD

**Discussion by Lobo-Feroz** [binarybottle/engram-es#11] [General]
Here's a list of the websites and software that could be used to learn touch typing with the engram-es layout.
This post will be edited to reflect the current status of each.

Websites:

- [type-fu.com](https://type-fu.com/) 
  - Engram V1 layout: [Engram-ES - V1 - 20210826.tfl.zip](https://github.com/binarybottle/engram-es/files/7062230/Engram-ES.-.V1.-.20210826.tfl.zip)
  - Dictionary: spanish 10K most common words (needs cleanup): [Spanish words - 10K - 20210824.tfd.zip](https://github.com/binarybottle/engram-es/files/7062233/Spanish.words.-.10K.-.20210824.tfd.zip)
  - Works only on chrome-based web browsers.
  - The author will add engram-es as a selectable keyboard in the app, and spanish as a language. I'll send him the files once we have a stable V1 version

- [colemakclub](https://gnusenpai.github.io/colemakclub/) - can create custom layout. they have several layouts already so probably they'll accept another one
- [keybr.com](https://www.keybr.com/) - mail sent asking how to propose a new custom layout
- [monkeytype.com](https://monkeytype.com/) - they have many different layouts including engram-en so probably they'll accept another one. I'll ask them via discord or mail

- [typingclub.com](https://www.typingclub.com/) - they have spanish lessons on qwerty. might or might not accept new layouts.

Software:

- [Klavaro](https://klavaro.sourceforge.io/en/index.html). Stand alone typing software with custom keyboard layout capabilities
- [KTouch](https://apps.kde.org/ktouch/). Part of the KDE Suite. Custom keyboard layout capabilities. Linux only (KDE tools can be run on windows but it's not easy).

**Comment by NickG13** [comment]
I learned touch typing in typingclub, and they are pretty good indeed! Hope they end up accepting the definitive layout

**Comment by binarybottle** [comment]
Thank you very much for the suggestions! Once the engram-es is finalized, it would be great to have engram and engram-es layouts entered into these for people to practice!

**Comment by Lobo-Feroz** [comment]
Updated the status on type-fu with Engram-es V1 beta layout and spanish dictionary. Link to files in the first post of this thread.

![imagen](https://user-images.githubusercontent.com/10242383/131022945-dff359d2-03e9-447a-8111-399e94b4e9db.png)

**Reply by binarybottle** [reply]
Thank you so much!  I will add the above image to the repo and notebook!
